snapcraft/internal/build_providers/errors.py

```diff
@@ -103,11 +103,11 @@ class ProviderCommunicationError(ProviderBaseError):
 
     fmt = (
         "An error occurred when trying to communicate with the "
-        "{provider_name!r} provider."
+        "{provider_name!r} provider: {message}."
```

Comment:
should that be `{message!r}` ?

Reply:
that would quote the entirety of message where instead we might want part of message to have quotes.